src/watch_test.ts

```diff
@@ -238,90 +211,28 @@ describe('Watch', () => {
                 receivedObjects.push(obj);
             },
             (err: any) => {
-                doneCalled = true;
                 doneErr.push(err);
+                doneResolve();
             },
         );
 
-        verify(fakeRequestor.webRequest(anything()));
-
-        const [opts] = capture(fakeRequestor.webRequest).last();
-        const reqOpts: request.OptionsWithUri = opts as request.OptionsWithUri;
+        stream.emit('error', errIn);
 
-        expect(reqOpts.uri).to.equal(`${server}${path}`);
```

Comment:
Are there equiavelent tests we could do for fetch?

Reply:
The fact that the test passes, means the correct uri was used, as this is the URL nock will listen on. If you change the URL for some reason, you will get an error from nock